include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -8160,86 +7837,15 @@ FEFaceEvaluation<dim,
   if (hessians_on_general_cells)
     evaluation_flag_actual |= EvaluationFlags::gradients;
 
-  if (this->dof_access_index ==
-        internal::MatrixFreeFunctions::DoFInfo::dof_access_cell &&
-      this->is_interior_face == false)
-    {
-      const auto face_nos          = this->compute_face_no_data();
-      const auto face_orientations = this->compute_face_orientations();
-
-#  ifdef DEBUG
-      // currently on structured meshes are supported -> face numbers and
-      // orientations have to be the same for all filled lanes
-      for (unsigned int v = 1; v < VectorizedArrayType::size(); ++v)
-        {
-          if (face_nos[v] != numbers::invalid_unsigned_int)
-            AssertDimension(face_nos[0], face_nos[v]);
-          if (face_orientations[v] != numbers::invalid_unsigned_int)
-            AssertDimension(face_orientations[0], face_orientations[v]);
-        }
-#  endif
-
-      internal::FEFaceEvaluationImplEvaluateSelector<dim, VectorizedArrayType>::
-        template run<fe_degree, n_q_points_1d>(
-          n_components,
-          *this->data,
-          values_array,
-          this->begin_values(),
-          this->begin_gradients(),
-          this->begin_hessians(),
-          this->scratch_data.begin(),
-          evaluation_flag_actual & EvaluationFlags::values,
-          evaluation_flag_actual & EvaluationFlags::gradients,
-          evaluation_flag_actual & EvaluationFlags::hessians,
-          face_nos[0],
-          this->subface_index,
-          face_orientations[0],
-          this->descriptor->face_orientations);
-    }
+  if (fe_degree > -1)
```

Comment:
Why `> -1` and not `>= 0` along with a variable of type `unsigned`? What is the meaning of negative degrees?